src/content/app/genome-browser/components/genome-browser-error/GenomeBrowserError.scss

```diff
@@ -0,0 +1,23 @@
+@import 'src/styles/common';
+
+.container {
+  --alert-icon-diameter: 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 56px;
+  row-gap: 34px;
+}
+
+.container p {
+  margin: 0;
+  text-align: center;
+}
+
+.container p + p {
```

Comment:
Curious about why the sibling selector is used in this way with the same element

Reply:
Ah gotcha. So it is an alternative to `:not(:first-child)`.

Reply:
True. I suppose `:not(:first-child)` would have been more expressive here.